### ISSUE TITLE
USWDS - Table: Scrollable tables are not fully accessible with screen readers

### ISSUE THREAD

**Issue by amyleadem** [uswds/uswds#5828]
## Summary
From @amycole501:
When using JAWS and NVDA I'm unable to get the left and right arrow keys to move the table left or right to read all the data. I can use a keyboard to access the scroll bar when NOT using assistive technology.

When in JAWS or NVDA mode, however, the arrow keys don't move the table left or right. The table data is still audible offscreen. It's possible the issue would only affect AT users who are sighted or AT users who have some vision and want to see which data cell the focus is on.

In any event we do need to ensure that the scroll functionality is working on the keyboard when using JAWS and NVDA.

**Comment by amycole501** [comment]
When using JAWS and NVDA I'm unable to get the left and right arrow keys to move the table left or right to read all the data. I can use a keyboard to access the scroll bar when NOT using assistive technology.

When in JAWS or NVDA mode, however, the arrow keys don't move the table left or right. The table data is still audible offscreen. It's possible the issue would only affect AT users who are sighted or AT users who have some vision and want to see which data cell the focus is on.

In any event we do need to ensure that the scroll functionality is working on the keyboard when using JAWS and NVDA.

**Comment by brunerae** [comment]
Need to confirm whether this is a known issue with the screen readers or whether its something on our end.

**Comment by brunerae** [comment]
@uswds/a11y - needs confirmation! Thank you!

**Comment by amycole501** [comment]
Two of us tested using NVDA and the arrow keys work. This appears to be an issue with JAWS.

One workaround is for the user to turn the virtual cursor and do some keyboard commands to get it to work. All that to say it's probable this is limited to JAWS and appears it may be a bug as reported here https://github.com/FreedomScientific/standards-support/issues/607

I don't think at this point we should not do further work on this as the table code doesn't seem to need modification.

**Comment by chandracarney** [comment]
In June, we discussed that this needs further testing and triage. Here are our notes.

Seems to be an issue in JAWS (rather than an issue with the component). 
- Can test using NVDA to double check. 
- After confirming, we will likely close the issue overall. 
- There is already an issue was filed with JAWS, if time, someone can follow up on the filed JAWS issue.

**Comment by ryparker** [comment]
The follow-up testing narrowed this to JAWS; NVDA worked in the maintainers' tests. FreedomScientific/standards-support#607 is still open, but it covers virtual-focus scrolling more generally and doesn't resolve this specific horizontal-table case.

Let's test the current table in JAWS with Chrome and compare it with NVDA. That should help us decide whether USWDS needs a change or whether we should document a JAWS limitation and workaround.
